.eslintrc.yml

```diff
@@ -38,9 +38,9 @@ settings:
         - .tsx
 rules:
   '@typescript-eslint/no-floating-promises': ['error']
-  no-shadow: 0
-  linebreak-style: 0
-  no-console: 0
+  no-shadow: off
+  linebreak-style: off
+  no-console: warn
```

Comment:
Reason for enabling no-console and adding eslint disable next line on some console logs?

Reply:
some console logs might be useful for debugging, so I disable the eslint warning for them